webapp/src/greenscreen.tsx

```diff
@@ -91,10 +91,6 @@ export class WebCam extends data.Component<WebCamProps, WebCamState> {
     private stop() {
         this.deviceId = undefined;
         if (this.stream) {
-            try {
-                if (this.stream.stop)
-                    this.stream.stop();
```

Comment:
this wasn't needed?

Reply:
oh strange, this might be a merge gone wrong. I'll investigate

Reply:
Ah, this was intentional. MediaStream.stop() was deprecated: https://developers.google.com/web/updates/2015/07/mediastream-deprecations

The right way is to stop the tracks which is what we do below